Edits to EPAS 13 - update clone schema setup steps #6336 (#6455)
EnterpriseDB/docs: product_docs/docs/epas/13/epas_guide/14_edb_clone_schema.mdx

The Log directory is required to store all the log files.

After creating the extensions, to create the log directory, execute the following statement as a superuser:

  ```sql
  SELECT edb_util.create_clone_log_dir();
  ```

> When successful, command returns `true`.

Author: ```suggestion
When successful, the command returns `true`.
```

**This foreign server and user mapping must be created within the database of the local server in which the cloning is to occur.**

> **When invoking an EDB Clone Schema function, the database user for whom the user mapping is defined must have the required privileges and be connected to the local server .**

Author: ```suggestion
**When invoking an EDB Clone Schema function, the database user for whom the user mapping is defined must have the required privileges and be connected to the local server.**
```

**All of these foreign servers and user mappings must be created within the target database of the target/local server.**

> **When invoking an EDB Clone Schema function, the database user for whom the user mappings are defined must have the required privileges and be connected to the local server .**

Author: ```suggestion
**When invoking an EDB Clone Schema function, the database user for whom the user mappings are defined must have the required privileges and be connected to the local server.**
```

### REVOKE_CLONE_SCHEMA_PRIVILEGES

> You can revoke the clone schema privileges from a non-superuser using this function.

Author: ```suggestion
This function revokes the clone schema privileges from a non-superuser using this function.
```
